### DISCUSSION TITLE
useHierarchicalMenu with rootPath problem

### DISCUSSION THREAD

**Discussion by SergiuGrisca** [algolia/instantsearch#5626] [Q&A]
hello, I have a problem here our useHierachicalMenu is not returning any items if I pass rootPath to it on the first render... 
on the second render (some filter change etc. it's ok) what may cause this, and what can be a possible fix ?

**Comment by Arya-Aghaei** [comment]
Hey, I also have the same problem, when I upgraded algolia packages, when I use `useHierarchicalMenu` and I call the `refine` function, `isRefined` flag of the selected item for refinement will not change anymore in the result items, I hope someone suggest any workaround for that.
Also I should mention if I remove the package-lock.json and install packages, this issue happens too.
